bliss/simulator/prior.py

```diff
@@ -222,7 +205,7 @@ def multiflux(self, flux_ratios, flux_base) -> Tensor:
 
         return total_flux
 
-    def _flux_ratios_against_b(self, survey_data_dir, image_ids, items, b) -> Tuple[dict, dict]:
+    def _flux_ratios_against_b(self, survey_data_dir) -> Tuple[dict, dict]:
         """Sample and compute all star, galaxy fluxes relative to `b`-band based on real image data.
 
         Instead of pareto-sampling fluxes for each band, we pareto-sample `b`-band flux values,
```

Comment:
This comment may be out-of-date as there's no `b` in scope

Reply:
A new function name maybe helpful too now that `b` isn't an argument

Reply:
I actually have b as an argument in `SDSSPrior` so I just added it back. Think all of the '{survey}Prior' will need that argument.

Reply:
The general use of this function is similar to what we want the `subclass.sample` functions to be. But I think it would be better to just get rid of this and add abstract `_sample_star_ratios` and `_sample_galaxy_ratios` as mentioned in my other comment.

Reply:
Actually taking this back ^, `b` shouldn't be an argument for this method since it's survey-specific.